.github/workflows/pythontests.yml

```
@@ -30,10 +30,23 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python -m pip install poetry
+          sed -i '/sphinx-antsibull-ext = /d' pyproject.toml
           poetry install
           poetry update
+          poetry run pip install sphinx-extension/
       - name: Test with pytest and upload coverage stats
         run: |
           ./test-pytest.sh
           poetry run coverage xml -i
           poetry run codecov --token=${{ secrets.CODECOV_TOKEN }}
+      - name: Install dependencies for Sphinx extension
+        run: |
+          poetry install
+          poetry update
+        working-directory: ./sphinx-extension
+      - name: Test with pytest and upload coverage stats
+        run: |
+          ./test-pytest.sh
+          poetry run coverage xml -i
+          poetry run codecov --token=${{ secrets.CODECOV_TOKEN }}
```

codecov doesn't need a token for GHA runs anymore.
```suggestion
          poetry run codecov
```